nixos/modules/services/misc/osticket/default.nix

```
@@ -0,0 +1,282 @@
+{ config, lib, pkgs, ... }:
+
+let
+  version = "1.15.4";
+  sha256 = "1nic4sg7n1q3fr4sj2j7xgv3a5p1gq4xccmx5msci1wwx62mfjfw";
+  user = "osticket";
+  stateDir = "/var/lib/${user}";
+
+  inherit (lib) mkDefault mkEnableOption mkForce mkIf mkMerge mkOption;
+  inherit (lib)
+    concatStringsSep literalExample mapAttrsToList optional optionals
+    optionalString types;
+  inherit (pkgs) fetchFromGitHub;
+
+  cfg = config.services.osticket;
+
+in
+{
+  # interface
+  options = {
+    services.osticket = {
+      enable = mkEnableOption "the osTicket issue tracker with mysql and phpfpm+nginx";
+
+      withSetup = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Include the setup wizard. Set this to false after setup.
+        '';
+      };
+
+      favicon = mkOption {
+        type = with types; nullOr path;
+        default = null;
+        description = ''
+          Path to custom favicon PNG image. This will be added to the page <HEAD> sections.
```

```suggestion
          Path to custom favicon PNG image. This will be added to the page &ltlHEAD&gt; sections.
```
that's probably the source of your manual troubles

---

thank you!